controllers/authorization/suite_test.go

```diff
@@ -32,42 +28,39 @@ var _ = SynchronizedBeforeSuite(func(ctx context.Context) {
 		return
 	}
 
-	testScheme := runtime.NewScheme()
-	pschema.RegisterSchemes(testScheme)
-	utilruntime.Must(apiextv1.AddToScheme(testScheme))
-
-	authzCtrl := authorization.NewPlatformAuthorizationReconciler(
-		nil, // SetupWithManager will ensure the client defined for the manager is propagated to the controller under test
-		ctrl.Log.WithName("controllers").WithName("platform"),
-		spi.AuthorizationComponent{
-			CustomResourceType: spi.ResourceSchema{
-				GroupVersionKind: schema.GroupVersionKind{Version: "v1", Kind: "service"},
-				Resources:        "services",
+	envTest, cancelFunc = test.StartWithControllers(
+		authorization.NewPlatformAuthorizationReconciler(
+			nil,
+			ctrl.Log.WithName("controllers").WithName("platform"),
+			spi.AuthorizationComponent{
+				CustomResourceType: spi.ResourceSchema{
+					GroupVersionKind: schema.GroupVersionKind{
+						Version: "v1",
+						Group:   "opendatahub.io",
+						Kind:    "Component",
+					},
+					Resources: "components",
+				},
+				WorkloadSelector: map[string]string{},
+				Ports:            []string{},
+				HostPaths:        []string{"spec.host"},
```

Comment:
Normally we would watch `metadata.annotations` or `.status` but I think for the sake of tests it is sufficient at the moment.